.github/workflows/rust.yml

```diff
@@ -30,6 +30,11 @@ jobs:
         id: detect-trigger
         with:
           keyword: "[test-df-upstream]"
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v1
+        with:
+          version: '3.x'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Don't think this should be necessary since this job isn't doing any compilation

```suggestion
```

Reply:
Actually this one needs to stay. Cargo Check runs a build command under the covers against ADP which in turn requires protoc to build out substrait proto files.